common/services/prismic/opening-times.js

```diff
@@ -0,0 +1,168 @@
+// @flow
+import Prismic from 'prismic-javascript';
+import {getMemoizedPrismicApi} from './api';
+import {isDatePast, london} from '../../utils/format-date';
+import groupBy from 'lodash.groupby';
+import type {ExceptionalVenueHours, PlacesOpeningHours, ExceptionalOpeningHoursDay, Venue, Days, OpeningTimes} from '../../model/opening-hours';
+import type {PrismicApiSearchResponse} from '../../services/prismic/types';
+import type Moment from 'moment';
+
+export async function getCollectionOpeningTimes() {
+  const prismic = await getMemoizedPrismicApi();
+  const collectionVenues = await prismic.query([
+    Prismic.Predicates.any('document.type', ['collection-venue'])
+  ]);
+
+  return parseVenuesToOpeningHours(collectionVenues);
```

Comment:
Why not use the functions here (sorry it's some conventions that have been forming, but not necessarily shared)?
https://github.com/wellcometrust/wellcomecollection.org/blob/master/common/services/prismic/api.js